status-im/status-react
# [#4871] Added some more DApps

fixes #4871

### Steps to test:
- Open Status
- Make sure newly added DApps are there

status: ready
